pusher-api/troubleshooting.md

```diff
@@ -0,0 +1,86 @@
+# Troubleshoot errors in the Pusher API
+* [Failing webhooks](#failing-webhooks)
+    * [Retry policy](#retry-policy)
+    * [Root cause](#root-cause)
+    * [Fix 404 DESTINATION_RESPONDED_WITH_ERROR_CODE](#fix-404-destination_responded_with_error_code)
+    * [Fix 400 DESTINATION_NOT_ACCESSIBLE](#fix-400-destination_not_accessible)
+* [Related API reference](#related-api-reference)
+
+## Failing webhooks
+When the destination URL does not reply with a successful HTTP status code (2xx), the Pusher API will mark the webhook as failing. Also, the Pusher API will notify you of the failing webhook at the email address that is used in the event subscription.
+
+### Retry policy
+The Pusher API will also re-send event notifications to the destination URL with retry attempts in the following interval and sequence:
+1. One retry attempt every 60 seconds for 10 attempts in total
+2. One retry attempt every 600 seconds for nine attempts in total
+3. One retry attempt every hour in the next 70 hours
+
+During the retry, when the destination URL starts responding with a successful status code, the subscription will be marked as active again.
+
+After the retry, if the destination URL still doesn't reply with a 2xx code, the event subscription will be deleted. You will need to create the event subscription again.
+
+### Root cause
+Failing webhooks may be caused by faulty destination URLs. One of the following error codes may return:
+
+* `HTTP 400 DESTINATION_NOT_ACCESSIBLE`  
+ 
+* `HTTP 404 DESTINATION_RESPONDED_WITH_ERROR_CODE`
+
+### Fix 404 DESTINATION_RESPONDED_WITH_ERROR_CODE
```

Comment:
The HTTP status code for `DESTINATION_RESPONDED_WITH_ERROR_CODE` is 500 (Internal Server Error) and not 404, so this header should say:

### Fix 500 DESTINATION_RESPONDED_WITH_ERROR_CODE

Reply:
Thanks Pär! I will change the header accordingly. 

However, when I tested the error in Postman before, I got this message:
´´´
"developerMessage": "Server responded with HTTP 404",
    "errorType": "DESTINATION_RESPONDED_WITH_ERROR_CODE",
    "violations": []
}
´´´

When I tried again today, I got this message:
```
{
    "developerMessage": "Server responded with HTTP 410",
    "errorType": "DESTINATION_RESPONDED_WITH_ERROR_CODE",
    "violations": []
}
```

Both of them has status code 500.